### PR TITLE
coc-254-Refactor-5.1

### PR DESCRIPTION
**Pull request by kdamas93**
https://github.com/2600hz/kazoo-teletype/pull/52

### CODE REVIEW

File: src/modules/cb_conferences.erl [hunk 1]
```diff
@@ -434,13 +434,18 @@ build_email_invite_kapi_req(Context) ->
...
+    ,{<<"Conference-Call-In-Numbers">>, maybe_remove_not_numeric_values([kz_json:get_binary_value(<<"numbers">>, CF) || CF <- Callflows])}
     ,{<<"Conference-Participant-Pins">>, kzd_conferences:member_pins(Conf)}
     ,{<<"Invite-Message">>, kz_json:get_binary_value(<<"invite_message">>, ReqData)}
     ,{<<"Guests">>, kz_json:get_list_value(<<"guests">>, ReqData)}
     | kz_api:default_headers(?APP_NAME, ?APP_VERSION)
     ].
 
+-spec maybe_remove_not_numeric_values(list()) -> list().
+maybe_remove_not_numeric_values(ConferenceCallInNumbers)->
+    ToFilterOut = [<<"undefinedconf">>],
```
Comment (jamesaimonetti) [inline src/modules/cb_conferences.erl:446]: If you only have the one value, hard-coded, you don't need to call lists:member below.
```erlang
[N || N <- ConferenceCallInNumbers, N =/= <<"undefinedconf">>].
```
Or `lists:delete(<<"undefinedconf">>, ConferenceCallInNumbers)` if only one instance of that value is present.

**Review by jamesaimonetti — CHANGES_REQUESTED** [review]

File: src/modules/cb_conferences.erl (hunk 1 above)
Comment (icehess) [reply src/modules/cb_conferences.erl:446]: Also move this logic to teletype itself. less code change and prs to merge.

Comment (icehess) [reply src/modules/cb_conferences.erl:446]: @jamesaimonetti is it safe to use `knm_converters:is_reconcilable/1` here? I don't think we ever use or even allow to use extension for conference?

Comment (jamesaimonetti) [reply src/modules/cb_conferences.erl:446]: There's no restriction on using extensions for a conference that I'm aware of.

**Comment by kdamas93** [comment]
Moving logic to teletype, less code change and prs to merge.

**Comment by kdamas93** [comment]
Moving logic to teletype, less code change and prs to merge.

**Pull request closed without merging**